test/src/test/java/jenkins/ClassPathTest.java

```diff
@@ -51,7 +52,7 @@ public void uniqueness() throws Exception {
         for (File jar : new File(WarExploder.getExplodedDir(), "WEB-INF/lib").listFiles((dir, name) -> name.endsWith(".jar"))) {
             String jarname = jar.getName();
             try (JarFile jf = new JarFile(jar)) {
-                Iterators.forEnumeration(jf.entries()).forEachRemaining(e -> {
+                Iterators.forEnumeration( jf.entries()).forEachRemaining( e -> {
```

Comment:
Looks like just whitespace changes here. Perhaps you meant something like this:

```java
for (JarEntry e : Collections.list(jf.entries())) {
```

Reply:
well this class has only one test and it;s marked as `@Ignored` I wanted to remove guava usage but then I gave up as it's not even executed. Anyway I will restore spaces...